rp235x-hal-examples/.cargo/config.toml

```diff
@@ -5,6 +5,33 @@
 # writing programs for Raspberry Silicon microcontrollers.
 #
 
+# Add aliases for building and running for the ARM and RISC-V targets.
+[alias]
+
+# Build arm or riscv using defaults or other options
```

Comment:
What does "using defaults or other options" mean in this context?
I think this information does not help the reader, it would be better to omit that text both here and in the run commands below